bears/go/GoVetBear.py

```diff
@@ -25,5 +28,96 @@ class GoVetBear:
     CAN_DETECT = {'Unused Code', 'Smell', 'Unreachable Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return 'vet', filename
+    def create_arguments(filename, file, config_file,
+                         assembly_go_mismatch: bool=False,
+                         useless_assignment: bool=False,
+                         atomic: bool=False,
```

Comment:
many things can be atomic ;)

Reply:
Actually, atomic here refers to the sync/atomic package in Go.
https://golang.org/pkg/sync/atomic/